src/GovUk.Education.ExploreEducationStatistics.Content.Services/ViewModels/KeyStatisticViewModels.cs

```diff
@@ -0,0 +1,44 @@
+#nullable enable
+using System;
+using System.Runtime.Serialization;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using JsonKnownTypes;
+using Newtonsoft.Json;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Services.ViewModels;
+
+[JsonConverter(typeof(JsonKnownTypesConverter<KeyStatisticViewModel>))]
+[JsonDiscriminator(Name = "type")]
+[KnownType(typeof(KeyStatisticTextViewModel))]
+[KnownType(typeof(KeyStatisticDataBlockViewModel))]
+public abstract record KeyStatisticViewModel
+{
+    public Guid Id { get; set; }
+
+    public string? Trend { get; set; }
+
+    public string? GuidanceTitle { get; set; }
+
+    public string? GuidanceText { get; set; }
+
+    public int Order { get; set; }
+
+    public DateTime Created { get; set; }
+
+    public DateTime? Updated { get; set; }
+}
+
+[JsonKnownThisType(nameof(KeyStatisticDataBlock))]
+
+public record KeyStatisticDataBlockViewModel : KeyStatisticViewModel
```

Comment:
There's a random blank line crept in here. Same problem in `Admin.ViewModels` too.

Reply:
I fixed that and forgot to push - pushed now :)